dvc/command/render.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import logging
+
+from dvc.command.base import CmdBase, append_doc_link
+from dvc.utils.serialize._yaml import dumps_yaml, render_dvc_template
+
+logger = logging.getLogger(__name__)
+
+
+class CmdRender(CmdBase):
```

Comment:
What use case do you see for a separate `render` command? I see that you've already implemented automatic rendering, so not quite sure what is the use case for a separate command.

Reply:
Thanks for the feedback!

Ahhh, this is mainly for a user to debug their `dvc.yaml`. Just to check if the rendered stage and variables are doing what you want them to.